app/controllers/account/users_controller.rb

```diff
@@ -0,0 +1,59 @@
+class Account::UsersController < Account::ApplicationController
+
+  def index
+    @users = collection
+  end
+
+  def show
+    @user = resource
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to account_user_path(id: @user.id)
+    else
+      render :new
+    end
+  end
+
+  def edit
+    @user = resource
+  end
+
+  def update
+    @user = resource
+    if @user.update_attributes(user_params)
+      redirect_to account_user_path(id: @user.id)
+    else
+      render 'edit'
```

Comment:
try avoid to user strings, when it is not nessasery, use symbols, `render :edit`